File: share-code/bai-tap-lap-trinh/cpp/32.cpp

#include <iostream>
using namespace std;

int main()
{
    int arrInt[] = {1, 2, 3, 4, 5};
    int avg = 0;
    int sum = 0;
    int count = 0;
    int length = sizeof(arrInt) / sizeof(int);

    for (int i = 0; i < length; i += 2)
    {
        if (arrInt[i] % 2 == 0)
        {
            sum += arrInt[i];
            count++;
        }
    }
    if (count > 0)
    {
        avg = sum / count;
    }
    avg = 0;
    cout << "count: " << count << endl;
    cout << "avg: " << avg << endl;
}

File: share-code/bai-tap-lap-trinh/python/array/32.py
arr = [2, 4, 6, 8, 8]

# Cách 1:
total_sum = 0
count = 0

for i in range(0, len(arr), 2):
    if arr[i] % 2 == 0:
        total_sum += arr[i]
        count += 1

avg = total_sum / count if count > 0 else 0

# Cách 2:
# selected_elements = [arr[i] for i in range(0, len(arr), 2) if arr[i] % 2 == 0]

# avg = sum(selected_elements) / len(selected_elements) if len(selected_elements) > 0 else 0

print("Avg:", avg)